src/main/java/org/folio/innreach/domain/entity/MARCTransformationOptionsSettings.java

```diff
@@ -0,0 +1,60 @@
+package org.folio.innreach.domain.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.folio.innreach.domain.entity.base.Auditable;
+import org.hibernate.annotations.Fetch;
+
+import javax.persistence.CascadeType;
+import javax.persistence.CollectionTable;
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "marc_transformation_options_settings")
+public class MARCTransformationOptionsSettings extends Auditable<String> {
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private UUID id;
+
+  @OneToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "central_server_id", unique = true)
+  private CentralServer centralServer;
+
+  @Column(name = "central_server_record_id")
+  private UUID centralServerRecordId;
```

Comment:
What's the purpose of this column?
If it's to represent "The central server record ID" from the requirements then I assume `private CentralServer centralServer` serves the same purpose